### PR TITLE
Added iOS and React Native code samples for setup intent

### PR DESCRIPTION
**Pull request by monica-m-ps**

### CODE REVIEW

File: mobile-sdk/ios/setup-intent/swift/setup-intent.swift [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import dojo_ios_sdk_drop_in_ui
+
+let dojoUI = DojoSDKDropInUI()
+dojoUI.startSetupFlow(paymentIntentId: "<paymentIntentId>", controller: self) {
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/ios/setup-intent/swift/setup-intent.swift:4]: `dojoUI.startSetupFlow(setupIntentId:"<setupIntentId>"`
we renamed paymentIntent to setupIntent for that call

File: mobile-sdk/ios/setup-intent/objectivec/setup-intent.m [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#import <dojo_ios_sdk_drop_in_ui-Swift.h>
+
+@property DojoSDKDropInUI *dojoUI;
+
+self.dojoUI = [[DojoSDKDropInUI alloc] init];
+[self.dojoUI startSetupFlowWithPaymentIntentId:@"<paymentIntentId>"
```
Comment (deniss-kaibagarovs) [inline mobile-sdk/ios/setup-intent/objectivec/setup-intent.m:6]: `startSetupFlowWithSetupIntentId` same here

**Review by deniss-kaibagarovs — CHANGES_REQUESTED** [review]

File: mobile-sdk/ios/setup-intent/swift/setup-intent.swift (hunk 1 above)
Comment (monica-m-ps) [reply mobile-sdk/ios/setup-intent/swift/setup-intent.swift]: Right. Fixed.

File: mobile-sdk/ios/setup-intent/objectivec/setup-intent.m (hunk 2 above)
Comment (monica-m-ps) [reply mobile-sdk/ios/setup-intent/objectivec/setup-intent.m]: Fixed

**Review by deniss-kaibagarovs — APPROVED** [review]

**Review by aaronforinton-dojo — APPROVED** [review]

**Pull request merged**